docs/UserGuide.adoc

```diff
@@ -273,37 +273,123 @@ Examples:
 
 * `set-theme <FILE_NAME>.xml`
 
-=== Add a new administrator account: `addAdmin` `[coming in v2.0]`
+// tag::admincommands[]
+=== Add a new administrator account: `addAdmin`
 
 Create and add a new adminstrator account. +
-Format: `addAdmin user\<USERNAME> pass\<PASSWORD> n\<NAME> ptp\<PATH_TO_PIC> salary\<SALARY> date\<EMPLOYMENTDATE>`
+Format: `addAdmin OPTION/INPUT...` +
 
-Examples:
+Input fields:
+
+* Username : `user/<USERNAME>`
+* Password : `pass/<PASSWORD>`
+* Name : `n/<NAME>`
+* Save path : `sp/<SAVE_PATH>`
+* Salary : `salary/<SALARY>`
+* Employment date : `date/<EMPLOYMENTDATE>`
+
+Example:
 
-* `addAdmin user\master pass\myPass n\John Doe ptp\johnDoe.img salary\4000 date\01/01/2018` +
+* `addAdmin user/master pass/myPass#1 n/John Doe sp/userconfig.xml salary/4000 date/01/01/2018` +
```

Comment:
It would be good to rename `sp/userconfig.xml` to `sp/userdata.xml` for clarity.

Reply:
Renamed.